src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -3,19 +3,28 @@
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 
+import edu.kis.powp.jobs2d.Figures;
 import edu.kis.powp.jobs2d.drivers.DriverManager;
+import edu.kis.powp.jobs2d.drivers.adapter.Job2dDriverAbstractDriverAdapter;
+import edu.kis.powp.jobs2d.magicpresets.FiguresJane;
 import edu.kis.powp.jobs2d.magicpresets.FiguresJoe;
 
 public class SelectTestFigureOptionListener implements ActionListener {
 
-	private DriverManager driverManager;
+    private DriverManager driverManager;
 
-	public SelectTestFigureOptionListener(DriverManager driverManager) {
-		this.driverManager = driverManager;
-	}
+    public SelectTestFigureOptionListener(DriverManager driverManager) {
+        this.driverManager = driverManager;
+    }
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.